test/test_exporter.py

```diff
@@ -316,9 +340,38 @@ def test_sum_delta(self, mock_post):
             headers=self._headers)
 
     @patch.object(requests.Session, 'post')
-    def test_sum_delta_non_monotonic(self, mock_post):
-        metrics_data = self._metrics_data_from_data(
-            [self._create_sum(250, monotonic=False)])
+    def test_sum_delta_non_monotonic_is_dropped(self, mock_post):
+        metrics_data = self._metrics_data_from_data([
+            self._create_sum(
+                10,
+                monotonic=False,
+                aggregation_temporality=AggregationTemporality.DELTA)])
+
+        exporter = DynatraceMetricsExporter()
+
+        result = exporter.export(metrics_data)
+
+        self.assertEqual(MetricExportResult.SUCCESS, result)
+        mock_post.assert_not_called()
+
+    @patch.object(requests.Session, 'post')
+    def test_sum_cumulative_monotonic_is_dropped(self, mock_post):
+        metrics_data = self._metrics_data_from_data([self._create_sum(10,
+                                                                      monotonic=True,
+                                                                      aggregation_temporality=AggregationTemporality.CUMULATIVE)])
+
+        exporter = DynatraceMetricsExporter()
+
+        result = exporter.export(metrics_data)
+
+        self.assertEqual(MetricExportResult.SUCCESS, result)
+        mock_post.assert_not_called()
+
+    @patch.object(requests.Session, 'post')
+    def test_sum_cumulative_non_monotonic_exported_as_gauge(self, mock_post):
```

Comment:
```suggestion
    def test_non_monotonic_cumulative_sum_exported_as_gauge(self, mock_post):
```
Its easier to read I think